src/main/java/seedu/duke/commons/core/Grade.java

```diff
@@ -0,0 +1,38 @@
+package seedu.duke.commons.core;
+
+//@@author richwill28
+public class Grade {

```

Comment:
Just for clarification. May I know why this is a normal class rather than an ENUM?

Reply:
Good catch, my initial intention was to use `enum` actually. Unfortunately, `A+`, `A-`, `B+`, etc. are not considered as valid `enum` so I decided to put all the grades in an array instead. I'm not 100% sure on this though, so you might want to help me double check.

Reply:
Here is a link to a post regarding how to do it. [Link](https://stackoverflow.com/questions/11445173/java-how-to-add-special-character-in-enum/11445245). But for readability sake it is probably more readable to leave it as how you have done.